src/Symfony/Component/Console/Application.php

```diff
@@ -1091,6 +1091,35 @@ public function setDefaultCommand($commandName, $isSingleCommand = false)
         return $this;
     }
 
+    /**
+     * Defines the "APP_ENV" and "APP_DEBUG" environment variables from --env and --no-debug argv options.
```

Comment:
Defines the "APP_ENV" and "APP_DEBUG" environment variables by consuming --env and --no-debug from the command line arguments.